bot.py

```
@@ -18,6 +18,11 @@
 from command_handler import handle_bot_command, handle_sum_day_command, handle_sum_hr_command # Import command handlers
 from firecrawl_handler import scrape_url_content # Import Firecrawl handler
 from apify_handler import scrape_twitter_content, is_twitter_url # Import Apify handler
+from error_handler import (
+    handle_discord_error, log_error_with_context, ErrorSeverity,
+    DiscordAPIError, ConfigurationError, safe_execute_async
```

Ruff (0.11.9)

`error_handler.DiscordAPIError` imported but unused

Remove unused import: `error_handler.DiscordAPIError`

(F401)